### PR TITLE
Add analytics with Matomo

### PR DESCRIPTION
**Pull request by sarathms**
Fixes ooni/probe#1064

image

### CODE REVIEW

File: renderer/pages/settings.js [hunk 1]
```diff
@@ -178,36 +178,51 @@ class Settings extends React.Component {
...
+                <NumberOption
+                  onConfigSet={this.reloadConfig}
+                  label={<Text>Websites tested (0 means all)</Text>}
+                  optionKey='nettests.websites_url_limit'
+                  config={config}
+                />
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<Text>Collect analytics data using Matomo</Text>}
```
Comment (hellais) [inline renderer/pages/settings.js:219]: This string is missing from the translations. Can we use a string which we already have in the mobile app which is already translated?

File: renderer/pages/settings.js [hunk 2]
```diff
@@ -178,36 +178,51 @@ class Settings extends React.Component {
...
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.IncludeIP' />}
+                  optionKey='sharing.include_ip'
+                  config={config} />
+
+                <NumberOption
+                  onConfigSet={this.reloadConfig}
+                  label={<Text>Websites tested (0 means all)</Text>}
```
Comment (hellais) [inline renderer/pages/settings.js:212]: This strings is also missing.

File: renderer/pages/settings.js [hunk 3]
```diff
@@ -178,36 +178,51 @@ class Settings extends React.Component {
...
+                  label={<FormattedMessage id='Settings.Sharing.IncludeNetwork' />}
+                  optionKey='sharing.include_asn'
+                  config={config}
+                />
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.IncludeCountryCode' />}
+                  optionKey='sharing.include_country'
+                  config={config}
```
Comment (hellais) [inline renderer/pages/settings.js:201]: While we are at it, let's also remove the ability to disable submitting the country code. See: https://github.com/ooni/probe/issues/1105

Comment (sarathms) [reply renderer/pages/settings.js:201]: removed

File: renderer/pages/settings.js (hunk 1 above)
Comment (agrabeli) [reply renderer/pages/settings.js:212]: @hellais We don't have a string for this in the mobile app either. 

Perhaps we can replace the copy of this string with:

Collect app usage statistics

File: renderer/pages/settings.js (hunk 2 above)
Comment (agrabeli) [reply renderer/pages/settings.js:205]: Oh my, this string hasn't been translated? :(

Comment (agrabeli) [reply renderer/pages/settings.js:205]: We can replace the copy with:

Number of tested websites (0 means all)

Comment (sarathms) [reply renderer/pages/settings.js]: added

File: renderer/pages/settings.js (hunk 1 above)
Comment (sarathms) [reply renderer/pages/settings.js]: added

**Pull request merged**